src/views/v2/tickets/submit/must-login.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Block: Tickets
+ * Submit Login
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/tickets/submit/login.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link http://m.tri.be/1amp
+ *
+ * @since TBD
+ *
+ * @version TBD
+ *
+ * @var bool $must_login If the login is required to purchase tickets.
+ */
+
+if ( empty( $must_login ) ) {
+	return;
+}
+
+?>
+<a class="tribe-common-c-btn tribe-common-c-btn--small" href="<?php echo esc_url( Tribe__Tickets__Tickets::get_login_url() ); ?>">
```

Comment:
Should we put this into a var that passes into the template instead? That would help towards the goal of reducing function dependencies that could change in the future.

Reply:
Nevermind, I think it's probably fine